spec/routing/work_packages_spec.rb

```diff
@@ -53,8 +52,7 @@
       .to route_to(controller: 'work_packages',
                    action: 'index',
                    project_id: '1',
-                   id: '2',
-                   state: 'overview')
+                   state: '2/overview')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.